### PR TITLE
Support for trigger feeds

### PR DESCRIPTION
**Pull request by Himavanth**
Fixes https://github.com/adobe/aio-cli-plugin-runtime/issues/160
The command
```wsk trigger create <trigger_name> --feed <feed_action_name> --param p1 v1```
reads the feed input, adds it to the trigger's annotations, and also creates the feed.
```wsk trigger delete <trigger_name>```
does a get on the trigger, checks the annotations for feed and if it exists, deletes the feed before deleting the trigger itself.
In wsk cli, feeds are dealt with through triggers only.

Openwhisk npm (js api) does not do this handling of feeds. It exposes feeds API and lets the user of the api handle them explicitly.
## Description
Considering there are other "value-add" functionalities like this on top of openwhisk js, now we have an sdk (aio-lib-runtime) that will work as a "value-add" layer between aio-cli-plugin-runtime and the openwhisk js api.
Some of the cases where we need value-added functionality are 

- https://github.com/adobe/aio-cli-plugin-runtime/issues/133
- https://github.com/adobe/aio-cli-plugin-runtime/issues/136
- https://github.com/adobe/aio-cli-plugin-runtime/issues/159

Eventually plugin code will get sparse, with only code related to the "UI" / "cmdline interaction" in it.

## How Has This Been Tested?
New tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "@adobe/aio-lib-core-config": "^1.2.4",
+    "@adobe/aio-lib-runtime": "0.0.2",
```
Comment (purplecabbage) [inline package.json:12]: Where is this? The repo is not public?

Comment (shazron) [reply package.json:12]: it's already public

File: src/commands/runtime/trigger/create.js [hunk 2]
```diff
@@ -45,6 +45,9 @@ class TriggerCreate extends RuntimeBaseCommand {
       if (Object.keys(annotationParams).length) {
         triggerParams.annotations = annotationParams
       }
+      if (flags.feed) {
```
Comment (shazron) [inline src/commands/runtime/trigger/create.js:48]: what about:
1. --feed-param
2. --trigger-param
(see `wsk trigger create --help`)

Comment (shazron) [reply src/commands/runtime/trigger/create.js:48]: Suggest pushing off the two flags above into a new PR, if needed. Seems to me these are fairly new in 2020, discussion here: https://github.com/apache/openwhisk-cli/issues/293#issuecomment-569975564

Comment (shazron) [reply src/commands/runtime/trigger/create.js:48]: I'm not even sure if they are supported in the wskdeploy manifest format even.

Comment (meryllblanchet) [reply src/commands/runtime/trigger/create.js:48]: The approach SGTM.

**Pull request merged**